src/foam/u2/crunch/CapabilityCardView.js

```diff
@@ -120,6 +133,11 @@ foam.CLASS({
           )
         ).then(ucj => {
           if ( ucj ) this.cjStatus = ucj.status;
+          if ( this.cjStatus === foam.nanos.crunch.CapabilityJunctionStatus.GRANTED ){
```

Comment:
Better to import foam.nanos.crunch.CapabilityJunctionStatus and then just do this.CapabilityJunctionStatus.GRANTED.